tests_integration/conftest.py

```diff
@@ -172,6 +173,36 @@ def _temporary_project_directory(
     return _temporary_project_directory
 
 
+@pytest.fixture
+def snow_templating(runner):
+    @contextmanager
+    def _templating(
+        template_source: Path,
+        variables: List[str],
+        copy_render_to: Optional[Path] = None,
```

Comment:
I'm not sure what is the purpose of this parameter. Can you explain?

Reply:
`copy_render_to` allow to copy rendered files into other directory, `snow init` requires to render files into new directory, e.g `with project_directory` creates tmp copy of projects, if I want to render `snowflake.yml` file I need render it into empty directory and copy it later to tmp project.

Reply:
I'm wondering if we need to use `snow init`. Can we template only the SQL script?

Reply:
We could by copying the project template and calling `snowflake.cli.api.rendering.project_templates.render_template_files(project_root, list_of_paths_relative_to_project_root, dict_with_variables)` directly

Reply:
⬆️ this could also solve the problem with `0.0.0-test-patched` invalid version error